utils/src/foreign_field.rs

```diff
@@ -178,21 +178,21 @@ mod tests {
         let bytes = SECP256K1_MOD;
         let big = BigUint::from_bytes_be(bytes);
         let fe = ForeignElement::<BaseField, 3>::from_be(bytes);
-        assert_eq!(fe.to_big(), big);
+        assert_eq!(fe.to_biguint(), big);
```

Comment:
I guess this is not part of this PR

Reply:
no it is not, it is part of the XOR update one but I needed some of those modifications to make the code more readable. In any case XOR update needs to be merged before, and then this PR will look cleaner